resources/event-hackathon/accessibility-tips-for-jupyter-notebooks.md

```diff
@@ -0,0 +1,94 @@
+# Accessibility Tips for Creating & Publishing Jupyter Notebooks
+
+By the [Notebooks for all](https://github.com/Iota-School/notebooks-for-all) team
+
+Draft 2: February 21, 2023
+
+---
+
+When writing & publishing Jupyter notebooks, the notebook editor has a huge influence on the content’s accessibility and there is not much an average user can do to fix that. Depending on the export method and content, notebooks can allow for different levels of accessibility for people using assistive technology. In running user tests, we’ve found that users encounter notebooks in a variety of formats:
+
+[The editable notebook format](https://jupyter.org/try-jupyter/lab?path=notebooks%2FIntro.ipynb) This is **inaccessible** and has many obstacles to navigating the UI or to edit cells
```

Comment:
Can the links open in a new tab? (in particular for the interactive notebook, I was unable to use the back button to navigate back to this page)